File: ws/villagomez/u2/ws24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Food.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.zoo.model;

/**
 *
 * @author Domenica Villagomez, CommitCrew, DCCO-ESPE
 */
public class Food {
    private int id;
    private String description;

    public Food(int id, String description) {
        this.id = id;
        this.description = description;
    }

    @Override
    public String toString() {
        return "Food{" + "id=" + id + ", description=" + description + '}';
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }
    
}

File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Food.js
class Food {

    constructor(id, description){
        this._id = id;
        this._description = description;
    }

    toString() {
        return `Food{id=${this._id}, description=${this._description}}`;
    }

    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get description() {
        return this._description;
    }

    set description(description) {
        this._description = description;
    }

}

module.exports = Food;